content/md/products-super-power/product-mass-actions.md

```diff
@@ -69,32 +71,56 @@ When you click on the notification, you access the report page of the action.
 :::
 
-# Edit products attributes
+# Edit or add products attributes values
 
-To ease the mass enrichment of products attributes values, you can edit the attributes values of your products thanks to the `Edit attributes` bulk action in the product grid. For example, you can set the brand of a selection of products to Akeneo.
+:::warning
+These actions don't add attribute to products, only the attributes belonging to the families of the selected products will be updated.
+:::
+
+:::info
+The changes will be done for the scope and the locale selected in the product grid. If you have several locales or scopes, you will have to do several bulk actions.
+:::
+
+:::tips
+If you select some [product models](/articles/what-about-products-variants.html#what-is-a-product-model) in the grid, the attributes will be updated on the product models and all its children variant products according to the [family variant](/articles/what-about-products-variants.html#what-is-a-family-variant) definition. For example, if the description (defined as `common attributes`) and the weight (defined for each variant product) is mass edited, the description will be updated on the product models and the weight on each child variant product of the selected product models.
+:::
+
+## Edit products attributes values
+
+To ease the mass enrichment of products attributes values, you can edit the attributes values of your products thanks to the `Edit attributes values` bulk action in the product grid. For example, you can set the brand of a selection of products to Akeneo.
 
 1.  Search for products in the grid with the appropriate filters
 2.  Select the products to edit: tick the products you want to add via the down arrow of the toolbar: `All` for all the products recorded, `All visible` for all visible products on the ongoing page, `none` for none of the products
 3.  Click on `Bulk Actions`
-4.  Select `Edit attributes` and then `Next`
-5.  Click on `Select attributes` to select the attributes you want to update (for example the collection and the brand)
+4.  Select `Edit attributes values` and then `Next`
+5.  Click on `Select attributes` to select the attributes you want to update (for example the brand)
 6.  Add values for the selected attributes
 ![Mass edit options](../img/Products_BulkActionEditAttribute.png)
 7.  Click on `Next`
 8.  Confirm the action with the `Confirm` button located in the middle of the screen
 9.  The attributes values will be updated in the selected products
 
-:::warning
-This action doesn't add attribute to products, only the attributes belonging to the families of the selected products will be edited.
-:::
+## Add products attributes values (_since 2.2_)
+
+To add products attributes values to the multivalued attributes, you can add attributes values to your products thanks to the `Add attributes values` bulk action in the product grid. For example, you can add a new collection "summer 2018" on a selection of products.
```

Comment:
"multivalued attributes" I don't know if it's very easy to undersatnt. What about talking about multiselect attribute?

Reply:
I change to "attributes with multiple values"